### PR TITLE
Added editor config file

### PR DESCRIPTION
**Pull request by VivekSingh777**

### CODE REVIEW

**Comment by VivekSingh777** [comment]
@MattIPv4

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# http://editorconfig.org
+
+root = true
+
+[*]
+indent_style = tabs
```
Comment (MattIPv4) [inline .editorconfig]: We use spaces in the project, not tabs.

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (VivekSingh777) [reply .editorconfig]: cool will fix in a moment

Comment (VivekSingh777) [reply .editorconfig]: @MattIPv4 Done

**Comment by MattIPv4** [comment]
Sorry, but I’m going to close this in favour of #24 — feel free to jump in there with any feedback or crack on with the eslint issue :)

**Comment by VivekSingh777** [comment]
@MattIPv4 Cool, Not an issue

**Pull request closed without merging**